sheet_cleaner/geocoding/merge.py

```diff
@@ -0,0 +1,42 @@
+"""Script that merges all geo_admin tabs exported to tsv into a single tsv file."""
+import pandas as pd
+import numpy as np
+
+want_cols = ["input", "latitude", "longitude", "geo_resolution", "location", "admin3", "admin2", "admin1", "country_new", "admin_id"]
+
+inputs = [
+    "/Users/timothe/Downloads/LATAM _ Open COVID-19 Working Group - geo_admin.tsv",
+    "/Users/timothe/Downloads/New York, USA _ Open COVID-19 Working Group - geo_admin.tsv",
+    "/Users/timothe/Downloads/Asia_Oceania _ Open COVID-19 Working Group - geo_admin.tsv",
+    "/Users/timothe/Downloads/Africa _ Open COVID-19 Working Group - geo_admin.tsv",    
```

Comment:
Is there value in parameterising this, or are you the only person who'll run this script?

Reply:
Ah damn sorry this was just a test script left in my repo, it should not be committed. Removed.